prow/gerrit/client/client.go

```diff
@@ -319,12 +320,11 @@ func (c *Client) Authenticate(cookiefilePath, tokenPath string) {
 	c.lock.Lock()
 	was, c.authentication = c.authentication, auth
 	c.lock.Unlock()
-	logrus.Info("Authenticating gerrit requests...")
```

Comment:
Nit: It would be much cleaner to separate this out into its own commit. The way it stands it's hard to understand the diff vs the commit message.

Reply:
ack